ruby/.rubocop.yml

```diff
@@ -13,6 +13,36 @@ AllCops:
     - !ruby/regexp /old_and_unused\.rb$/
   NewCops: enable
 
+# Configure cops for styles that we enforce and autocorrect for (where applicable)
+# Please add new configurations in alphabetical order and make sure their keys are not re-used in the section below
+
+Layout/ExtraSpacing:
+  Enabled: true
+  AllowForAlignment: true
+  AllowBeforeTrailingComments: true
+  ForceEqualSignAlignment: true
+
+Layout/FirstArrayElementIndentation:
+  EnforcedStyle: consistent
+
+Layout/FirstHashElementIndentation:
+  EnforcedStyle: consistent
+
+Layout/FirstHashElementLineBreak:
+  Enabled: true
+
+Layout/HashAlignment:
+  Enabled: true
+  EnforcedHashRocketStyle: table
+  EnforcedColonStyle: table
+  EnforcedLastArgumentHashStyle: always_inspect
```

Comment:
What does this do?

Reply:
This enables the alignment of hash key/value pairs for hashes that use either rocket or colon-style separators.

The last directive includes the application of this styling to hashes that appear as the last argument of a method call (whether or not the hash in that position explicitly is surrounded by braces or is implicitly omitting the surrounding braces).